mon-pix/tests/integration/components/data-protection-policy-information-banner-test.js

```diff
@@ -89,7 +89,7 @@ module('Integration | Component | data-protection-policy-information-banner', fu
         test('displays the data protection policy banner', async function (assert) {
           // given
           stubCurrentUserService(this.owner, { shouldSeeDataProtectionPolicyInformationBanner: true });
-          _stubWindowLocationHostname('pix.fr');
+          _stubWindowLocationHref('https://pix.fr');
```

Comment:
Au lieu de passer par une fonction, on peut directement écrire:

```suggestion
          sinon.stub(Location, 'getHref').returns('https://pix.fr');
```